src/planning.ts

```diff
@@ -14,6 +14,7 @@ export interface PlanOptions {
 
   penUpHeight: number;
   penDownHeight: number;
+  penStrokeWidth: number;
```

Comment:
Arguably this value shouldn't be part of `PlanOptions`, since it doesn't affect planning. Perhaps it could go in `initialState` instead, alongside e.g. `progress`. The downside then is that the option wouldn't be persisted across page loads. I'm OK with having it here though, just food for thought.